src/vos/vos_tree.c

```diff
@@ -440,6 +446,14 @@ ktr_rec_free(struct btr_instance *tins, struct btr_record *rec, void *args)
 	krec = vos_rec2krec(tins, rec);
 	umem_attr_get(&tins->ti_umm, &uma);
 
+	vos_dtx_degister_record(&tins->ti_umm, krec->kr_dtx,
+				rec->rec_mmid, DTX_RT_KEY);
```

Comment:
you do not need check degister return value?

Reply:
No, the vos_dtx_degister_record() return nothing.

Reply:
ok